### DISCUSSION TITLE
Sometimes ABL Extension java process keeps doing something and takes loads of CPU

### DISCUSSION THREAD

**Discussion by ArthurBir** [vscode-abl/vscode-abl#60] [Q&A] [answered]
Not sure why, but for some reason java process which I guess is responsible for ABL Extension takes loads of CPU even when no programs are being built in a project. Any suggestions why this could happen? 

![image](https://user-images.githubusercontent.com/16022199/223371532-1d1547a2-b6c2-47e4-a401-2723d32979d1.png)

Really good project, btw! Keep it up! 👍

**Comment by ArthurBir** [comment]
Also, here's a full line of Command Line:

```
java --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.math=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.util.concurrent=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.text=ALL-UNNAMED -Dorg.slf4j.simpleLogger.showLogName=false -Dorg.slf4j.simpleLogger.defaultLogLevel=INFO -jar c:\Users\abirbalas\.vscode\extensions\riversidesoftware.openedge-abl-lsp-1.4.6\resources\abl-lsp.jar	Disabled
```

**Comment by gquerret** [comment, marked as answer]
Does that always happen on this project ? On all projects ? Or does that happen randomly ?
You can enable verbose logging, maybe that will provide a hint:
```
"abl.langServerDebug": true
```
